database/migrations/2024_06_14_201056_update_campaigns_add_deleted_by_deleted_at.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('campaigns', function (Blueprint $table) {
+            $table->softDeletes();
+            $table->index('deleted_at');
+            $table->foreign('deleted_by')->references('id')->on('users')->nullOnDelete();
```

Comment:
I don't see deleted_by created anywhere. Does laravel create it on its own with softDeletes ? 🤔